src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderSentenceResource.java

```diff
@@ -194,6 +196,7 @@ public List<OffenderSentenceDetail> postOffenderSentences(@RequestBody @Paramete
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "The list of offenders is returned.")})
     @Operation(summary = "Retrieves list of offenders (with associated sentence detail) - POST version using booking id lists.", description = "Retrieves list of offenders (with associated sentence detail) - POST version using booking id lists.")
+    @ProgrammaticAuthorisation("Access is determined by annotation on the service")
```

Comment:
Sometimes the service annotation cant be moved as it results in a different error message